src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -14,6 +14,7 @@
     <DebugSymbols>true</DebugSymbols>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ProjectGuid>{9E50E7BF-CD6E-4269-A6DD-59FD0BD6C0FD}</ProjectGuid>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">DNXCore50</PackageTargetFramework>
```

Comment:
Right now it needs to be DNXCore50 to be consistent with the internal package.  As a separate follow up we need:
1. Consolidate the facades into a single build as dotnet*.
2. Package System.Private.ServiceModel as dotnet* with a netcore50 specific build for UWP.